pkg/domain/infra/abi/play.go

```diff
@@ -594,6 +618,62 @@ func kubeContainerToCreateConfig(ctx context.Context, containerYAML v1.Container
 	return &containerConfig, nil
 }
 
+// getConfigMap returns a kubernetes configMap obtained from --configmap flag
+func getConfigMap(r io.Reader) (v1.ConfigMap, error) {
+	var cm v1.ConfigMap
+
+	content, err := ioutil.ReadAll(r)
+	if err != nil {
+		return cm, err
```

Comment:
Can you wrap this error so it's clear exactly where it's happening?

Reply:
done